R/class.R

```diff
@@ -0,0 +1,43 @@
+inject_super <- function(fun) {
+  e <- new.env()
```

Comment:
nit: prefer explicitly setting `parent = parent.frame()` to make it clear that this environment should inherit from the parent frame?

Reply:
Or should this have `environment(fun)` as the parent environment?

Reply:
you are totally right! fixed